app/helpers/discord.ts

```diff
@@ -56,6 +57,28 @@ export const quoteMessageContent = (content: string) => {
   return `> ${content.replace("`", "\\`").replace(/[\n]/g, "\n> ")}`;
 };
 
+/*
+ * Create a message embed that
+ */
+export const describeAttachments = (attachments: Message["attachments"]) => {
+  return attachments.size === 0
+    ? ""
+    : "Attachments:\n" +
+        attachments
+          .map(
+            (a) =>
```

Comment:
Same as above - IMO meaningless variable name.

Reply:
Broadly, I don't particularly mind meaningless names in simple functions like mappers/filters/etc, they're low enough context that I think it's okay. In this case though, I'm going to push up another commit that destructures the arg because it wraps better